tasks/GPSDTask.cpp

```diff
@@ -1,4 +1,14 @@
-#include "GPSDTask.hpp"
+
+// workaround for name clash in 
+// usr/include/mach/policy.h and gpsd/3.15/include/gps.h
```

Comment:
You could move it there but the problem only occurs in combination with libgpsmm.h which might not be included by other orogen tasks inherit from this task. Therefore, to not globally undef policy_t I would keep it in the c++ file, No?